app/models/mixins/reportable_mixin.rb

```diff
@@ -11,6 +11,10 @@ def sortable?
       true
     end
 
+    def scope_by_tenant?
+      false
```

Comment:
I had expected `scope_by_tenant?` type functionality to live in RBAC (like it seems to for other cases) and not the individual classes/mixins.